local-modules/ui-header/components/avatar.module.less

```diff
@@ -13,18 +13,19 @@
 	position: relative;
 	border: 1px solid @sk_white;
 	border-radius: 5px;
+	box-sizing: border-box;
```

Comment:
Oops! The changes to this file ended up attributed to the wrong commit. (The original change was made in a different non-public repo and got shunted over here via semi-automatic scriptery which I did not sufficiently double-check!)